xvelmap/xr_accessor.py

```diff
@@ -77,6 +85,15 @@ def plot(self, u_var, v_var, lat_dim='latitude', lon_dim='longitude',
 
         if units is None:
             units = ''
+            
+        # Prefers data to be in gaussian grid format (latitudes descending)
+        # This probably happens in '.flatten().tolist()'
+        if np.any(np.diff(self._ds[lat_dim].values) >= 0):
+            self._ds.coords[lat_dim] = self._ds.coords[lat_dim].values[::-1]
+            # Assumes latitude is the first axis.
+            # Could probably add a check here
+            for var_name in (u_var, v_var):
+                self._ds[var_name].values = self._ds[var_name].values[::-1,:]
```

Comment:
You can just use `self._ds.sel(lat_dim=slice(None, None, -1)` here. It will flip the coordinate and all related variables (no check is needed).

Reply:
I couldn't quite get this to work if lat_dim is not specified it will assume it is latitude.
It gives the error message ValueError: dimensions or multi-index levels ['lat_dim'] do not exist.
I could use .rename() on _ds to make lat_dim and lon_dim 'latitude' and 'longitude' explicitly.
I could then write _ds.sel(latitude=slice(None, None, -1))